packages/common/src/audius-query/hooks/usePaginatedQuery.ts

```diff
@@ -99,6 +99,7 @@ export const useAllPaginatedQuery = <
         setLoadingMore(false)
         return
       }
+      // what if we just selected the data instead of accumulating it
```

Comment:
🤔

Reply:
oh this is an old comment; ended up doing this